docs/api-reference/web-mercator-utils.md

```diff
@@ -175,3 +200,20 @@ Parameters:
 
 Returns:
 - `{longitude, latitude, zoom}`
+
+
+### `getMapCenterByLngLatPosition(opts)`
+
+Returns the map center that place a given [lng, lat] coordinate at screen point [x, y].Returns the map center that place a given [lng, lat] coordinate at screen point [x, y].
```

Comment:
Same sentence twice?
 
Maybe not for this PR but would be nice with a "Usage" section with a code example. Showing how this can be used to recenter screen on an object. 

Maybe some comment about how to obtain the unprojection matrix.